packages/base/src/dialogs/ProcessingFormDialog.tsx

```diff
@@ -37,19 +38,25 @@ export interface IProcessingFormWrapperProps
 const ProcessingFormWrapper: React.FC<IProcessingFormWrapperProps> = props => {
   const [ready, setReady] = React.useState<boolean>(false);
 
-  const okSignal = React.useRef<Signal<Dialog<any>, number>>();
-  const formErrorSignal = React.useRef<Signal<Dialog<any>, boolean>>();
+  const okSignal = React.useRef<Signal<Dialog<any>, number> | undefined>(
+    undefined,
+  );
+  const formErrorSignal = React.useRef<
+    Signal<Dialog<any>, boolean> | undefined
+  >(undefined);
```

Comment:
These refs are pretty well-typed already. `useRef`'s default for `initialValue` is `undefined`.